department-of-veterans-affairs/vets-website
# Income and asset 86445 annuities

## Summary
Import the 'Annuities' section of the PDF into the digital Income and Assets Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86445

## Related issue(s)
n/a

## Associated Pull Request(s)
n/a

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser

## How to verify
1. Continue to the Annuities chapter `/annuities-summary` 
2. Add an annuity
3. Verify the annuity can be edited and deleted

## What areas of the site does it impact?
Income and Asset Statement Form

## Screenshots
### Summary Page
| No Items | Item Added |
| ----------- | ----------- | 
|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities-summary](https://github.com/user-attachments/assets/64761472-ea25-4a2d-80f9-334b95779a30)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (6)](https://github.com/user-attachments/assets/ce2005ba-30d8-4745-997b-8a5af2bcbcb3)

### Item Pages
| Item Page | Item Page | Item Page | Item Page | Item Page | Item Page |
| ----------- | ----------- | ----------- | ----------- |  ----------- | ----------- | 
|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true](https://github.com/user-attachments/assets/e8742766-e9f7-4ef6-8e6a-851e2a9a6f49)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (1)](https://github.com/user-attachments/assets/9204b6a6-182e-41f7-b9bf-fc4e836fa014)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (2)](https://github.com/user-attachments/assets/179ad5d6-975a-4bf2-97f0-8aebfce2535a)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (3)](https://github.com/user-attachments/assets/106714d4-0925-4aee-aae8-8751a9d85df3)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (4)](https://github.com/user-attachments/assets/b33f54a5-d1a3-4040-a48f-dbe765be4871)|![localhost_3001_income-and-asset-statement-form-21p-0969_annuities_0_information_add=true (5)](https://github.com/user-attachments/assets/2ba0c699-98b8-48d2-9596-df77d6581510)





### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
